iron-list.html

```diff
@@ -729,6 +775,8 @@
 
       // increase the pool of physical items
       this._increasePoolIfNeeded();
+
+      this.fire('iron-list-dom-updated', null, {bubles: false});

```

Comment:
Curious what the impetus of adding this event was?  Have users asked for it?

FWIW, `dom-repeat` and `dom-if` both have similar events that are fired called `dom-change`; there's an opportunity to make the API's more similar by using that name (although `dom-change` bubbles, but given a 2nd chance at the API we'd make it not bubble like this for performance reasons).

Either way, it's missing event documentation. Can you add that (and consider the name change)?